### PR TITLE
Neos package fixes

### PR DESCRIPTION
**Pull request by anhiga**
Final fixes in `poliastro.neos` package before 0.7.0 release.
I've decided to keep the list of orbits as `orbit_from_name` return type, given the characteristics of DASTCOM5 database.

### CODE REVIEW

**Comment by Juanlu001** [comment]
The continuous integration is broken, unfortunately... Bad day to do a release, but it's what we have. We will try to trigger it tomorrow and get this in for the final 0.7.0.

https://status.github.com/

**Comment by anhiga** [comment]
I have a problem with really big legends, but I'm comfortable with the rest of the notebook.

**Comment by Juanlu001** [comment]
The CI failures are unrelated. Trying to fix them ASAP.

**Comment by Juanlu001** [comment]
This now requires rebase.

File: docs/source/dastcom5 parameters.rst [hunk 1]
```diff
@@ -0,0 +1,212 @@
+Dastcom5 parameters
+===================
```
Comment (Juanlu001) [inline docs/source/dastcom5 parameters.rst]: This is going to be super useful!

File: src/poliastro/neos/dastcom5.py [hunk 2]
```diff
@@ -383,13 +383,37 @@ def record_from_name(name):
...
     """
-    idx_path = os.path.join(DBS_LOCAL_PATH, 'dastcom.idx')
     records = []
+    lines = string_record_from_name(name)
+    for line in lines:
+        records.append(int(line[:6].lstrip()))
+    return records
+
+
+def string_record_from_name(name):
```
Comment (Juanlu001) [inline src/poliastro/neos/dastcom5.py]: Nice refactoring.

File: docs/source/examples/Using NEOS package.ipynb [hunk 3]
```diff
@@ -0,0 +1,1777 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Neos package\n",
+    "If you are interested in a given neo\n",
+    "The `poliastro.neos` package contains two modules:\n",
+    "* `dastcom5`:\n",
```
Comment (Juanlu001) [inline docs/source/examples/Using NEOS package.ipynb]: Is this WIP?

File: docs/source/examples/Using NEOS package.ipynb [hunk 4]
```diff
@@ -0,0 +1,1777 @@
...
...T5WZmZlxMzMzlb29fX12drZRnz59atasWeO5ZMkS\n8eTJk8sjIiIqH+X3RUhXoRYa0UtvvPFG/p49exyrqqqa/oajoqK8li5dWnDz5s2kzZs3ZyqVyqZl\nJiYmHACEQiGMjIy4QKBdJBAIoFKpGOec+fn51aSkpCSlpKQk3bx5M+mXX35JA4AzZ86k/elPfyqM\ni4uzCAsLk9XX31uG7P3333dxdnauT05OToqPj0+qr69vddzGY6tUKtbRc+Scsw0bNmQ1xpSTkxPf\n2Fpi7N7dtHzdUW3FFxoaqvztt9+SQkJCatasWSNetWqV2+/aOSFdjBIa0UsuLi7qiRMnlu7Zs8ex\n8T2FQiH08vKqB4Bt27Y5PMz+QkNDa0tKSoxOnjxpAQBKpZJduXLFVK1WIyMjQzRx4kTFp59+mq1Q\nKITl5eX3dHWWl5cL3dzc6oVCIT799FOHxvtZ7enVq1dtTk6OKDEx0QQAoqOj7dtab9y4ceWfffaZ\nU+M9vhs3bphUVFQIAG2XY0pKikitViMmJsZ+2LBhimHDhlXFxsZa5ebmGqlUKnz77bf2I0eOrBw5\ncmTVpUuXrBq7K/Pz8+87479cLje2srLSLF26tGT58uX5165dM+/gj5EQnaIuR6K31qxZk7d9+3an\nZq/vzp4929fFxaWuX79+VVlZWSb32745U1NTHh0dnbFs2TIvhUIhVKvVbMmSJfkhISHKOXPm+CgU\nCiHnnC1evDjf0dHxnoy1fPnygmnTpvkeOHDAbujQoYrm3aBtMTc35x9//HHmhAkT/Ozt7VUDBgyo\nTE5ONmu53ooVK4rkcrlJSEhIIOec2dvb1x89ejQDAHr16lX56quveqSkpJgNGDBA8cILL5QJhUK8\n9dZbOSNGjJByztnYsWPL/vCHP5QBwEcffSSfMmWKn0ajgYODQ/2FCxfS2osvLi7O7I033vAQCAQw\nMjLin376aav7hoR0R22Wj7l+/bo8LCysSAfxEEIe4MiRI1YbNmxwOXPmTLquYyFEV65fv+4YFhbm\n3fw96nIkhBBiECihEaJnJkyYoKDWGSGtUUIjhBBiECihEUIIMQiU0AghhBgESmiEEEIMQrdMaOnp\n6cYDBgyQ+vr6Bvn5+QX9/e9/d25clp+fLxw8eLC/RCIJHjx4sH9hYaEQAK5evWraq1
```
Comment (Juanlu001) [inline docs/source/examples/Using NEOS package.ipynb]: I wonder if we should take integers instead of strings here... what do you think?

Comment (Juanlu001) [inline docs/source/examples/Using NEOS package.ipynb]: You see?? The plotting of the apocenter is awful "thanks" to the sampling function in `plotting`.

**Comment by anhiga** [comment]
This is how plots look now:
figure_1-1

**Comment by Juanlu001** [comment]
I like how the plots look like now :) Ping me when this is ready to merge.

**Pull request merged**